test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -154,3 +154,20 @@ func @torch.aten.arange.start() -> !torch.vtensor<[?],si64> {
   %0 = torch.aten.arange.start %int0, %int10, %none, %none, %none, %none : !torch.int, !torch.int, !torch.none, !torch.none, !torch.none, !torch.none -> !torch.vtensor<[?],si64>
   return %0 : !torch.vtensor<[?],si64>
 }
+
+// ----
+// CHECK-LABEL: func @decompose.aten.max.dim
+// CHECK-SAME:              (%[[INP:.*]]: !torch.vtensor<[?,?,?],f64>) -> !torch.vtensor<[?,1,?],si64> {
+// CHECK:      %[[CST1:.*]] = torch.constant.int 1
+// CHECK:      %[[TRUE:.*]] = torch.constant.bool true
+// CHECK:      %[[MAX_DIM:.*]] = torch.custom.aten.max.dim %[[INP]], %[[CST1]], %[[TRUE]] :
+// CHECK-SAME: !torch.vtensor<[?,?,?],f64>, !torch.int, !torch.bool -> !torch.vtensor<[?,1,?],f64>
+// CHECK:      %[[ARG_MAX:.*]] = torch.aten.argmax %[[INP]], %[[CST1]], %[[TRUE]] : 
+// CHECK-SAME: !torch.vtensor<[?,?,?],f64>, !torch.int, !torch.bool -> !torch.vtensor<[?,1,?],si64>
+// CHECK:      return %[[ARG_MAX]] : !torch.vtensor<[?,1,?],si64>
+func @decompose.aten.max.dim(%arg0: !torch.vtensor<[?,?,?],f64>) -> !torch.vtensor<[?,1,?],si64> {
```

Comment:
nit: the convention in this file is to name the function after the op that is being decomposed. So it should be: `@torch.aten.max.dim`